auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java

```diff
@@ -0,0 +1,29 @@
+package org.wildfly.security.auth.util;
+
+import org.junit.Test;
+import org.wildfly.security.auth.util.RegexNameRewriter;
```

Comment:
@mittalaishwarya Just a minor,  this class is in the same package as this test. So you don't have to import it